app/models/manageiq/providers/ibm_cloud/object_storage/storage_manager.rb

```diff
@@ -182,9 +191,10 @@ def self.verify_iam(crn, api_key)
 
     begin
       resource_controller_api.get_resource_instance(:id => crn).result
-    rescue IbmCloudResourceController::ApiException => err
-      _log.error("GUID resource lookup failed: #{err.code} #{err.error}")
-      raise MiqException::MiqInvalidCredentialsError, err.error
+    rescue Exception => err
+      error_message = "CRN resource lookup failed: #{err.code} #{err.error}"
```

Comment:
If we are going to rescue the more general `Exception` class then we can't count on having `:code` or `:error`
When this happens what exception is raised?

Reply:
Oh yeah, this is kinda dangerous - is there something more specific we can capture?

Reply:
Should have used `IBMCloudSdkCore::ApiException` (force pushed a fix)